srcpkgs/exa/template

```diff
@@ -15,9 +17,9 @@ checksum="ff0fa0bfc4edef8bdbbb3cabe6fdbd5481a71abbbcc2159f402dea515353ae7c
  531596a1ef2a757c7728087529528150e6eb52bb8224fe575aa00a5f1b762849"
 
 post_extract() {
-	mkdir -p accoutrements
-	mv ../man accoutrements/
-	mv ../completions accoutrements/
+	mkdir -p "${build_wrksrc}/accoutrements"
```

Comment:
```suggestion
	mkdir "${build_wrksrc}/accoutrements"
```
post_extract is guaranteed to only run once, so -p is not required

Reply:
Done